README.md

```diff
@@ -50,7 +50,7 @@ Typically you'll use this in your [npm scripts][npm scripts] (or [package script
 ```json
 {
   "scripts": {
-    "format": "prettier-eslint src/**/*.js"
+    "format": "prettier-eslint 'src/**/*.js'"
```

Comment:
This wont work on windows. Could you change it to escaped double quotes? Also, if you like, you can [add yourself to the contributors list](https://github.com/kentcdodds/prettier-eslint-cli/blob/master/CONTRIBUTING.md#add-yourself-as-a-contributor) :)